keyboards/jlw/deskcheck/info.json

```
@@ -0,0 +1,125 @@
+{
+    "keyboard_name": "deskcheck",
+    "manufacturer": "jlw",
+    "url": "https://github.com/josh-l-wang/deskcheck",
+    "maintainer": "jlw",
+    "usb": {
+        "vid": "0x1209",
+        "pid": "0xA460",
+        "device_version": "0.0.1"
+    },
+    "processor": "STM32F072",
+    "bootloader": "stm32-dfu",
+    "matrix_pins": {
+        "direct": [ [ "B12", "A2", "A3", "A4" ] ]
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+    },
+    "ws2812": {
+        "driver": "bitbang",
+        "pin": "B4"
+    },
+    "rgb_matrix": {
+        "animations": {
+            "alphas_mods": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_sat": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "band_val": true,
+            "breathing": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "cycle_up_down": true,
+            "digital_rain": true,
+            "dual_beacon": true,
+            "gradient_left_right": true,
```

```suggestion
            "flower_blooming": true,
            "gradient_left_right": true,
```
new animation if you want (there are a couple of new ones)